README.md

```diff
@@ -698,15 +700,15 @@ The second line of output should equal 4
 ```
 
 
-shellspec supports to improve readability *language chains* like chai.js.
-It is only improve readability, does not any effect the expectation.
+Shellspec supports *language chains* like [chai.js](https://www.chaijs.com/).
```

Comment:
Thanks for adding the chai.js link. 😄

Reply:
I didn't know that library. It may make more people know about it.